terraform/modules/jupyter-s3-storage/jupyterhub_s3.tf

```diff
@@ -0,0 +1,45 @@
+resource "random_id" "jupyter_bucket" {
+  byte_length = 16
+}
+
+resource "aws_s3_bucket" "jupyter_storage" {
+  bucket = random_id.jupyter_bucket.hex
+  acl    = "private"
+
+  tags = var.common_tags
+
+  versioning {
+    enabled = true
+  }
+
+  logging {
+    target_bucket = var.logging_bucket
+    target_prefix = "S3Logs/${random_id.jupyter_bucket.hex}-ci/ServerLogs"
```

Comment:
Should this have a `ci` suffix?

Reply:
I was just following how other buckets had been set up in DIP - I can remove it, if you think that it is not needed?